rero_ils/modules/items/api/circulation.py

```diff
@@ -583,6 +592,11 @@ def extend_loan(self, current_loan, **kwargs):
     @add_action_parameters_and_flush_indexes
     def request(self, current_loan, **kwargs):
         """Request item for the user and create notifications."""
+        # Check if the 'checkout' action is allowed
```

Comment:
checkout or request?

Reply:
copy/paste and ICC problem ;-)
Corrected in next PR (avoiding rebase/cherry-pick....)